pkg/upgrade/container/container.go

```diff
@@ -37,6 +37,12 @@ func WithSecurityContext(securityContext *corev1.SecurityContext) Option {
 	}
 }
 
+func WithSELinuxOptions(seLinuxoptions *corev1.SELinuxOptions) Option {
```

Comment:
I'm not sure this is necessary - you're setting `container.SecurityContext.SELinuxOptions`, when `container.SecurityContext` is already set by the `WithSecurityContext` function. Just set this directly on the SecurityContext.